linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -58,8 +58,8 @@ private Expressions() {}
    * extra type information in generics.
    */
   public static String toString(List<? extends Node> expressions, String sep,
-      boolean generics) {
-    final ExpressionWriter writer = new ExpressionWriter(generics);
+      boolean generics, boolean methodSplit) {
```

Comment:
Probably the javadoc of this method should be updated to mention the new "optional split"

Reply:
Done in [dc1c66d](https://github.com/apache/calcite/pull/3901/commits/dc1c66d14b38ad122312b99861fca20794701504)